src/frontend/js/buttons-icons.js

```diff
@@ -0,0 +1,46 @@
+import { LitElement, html } from "lit";
+export class buttons_icons extends LitElement {
+  static get properties() {
+    return {
+      content: "",
+    };
+  }
+  constructor() {
+    super();
+    this.content = "";
+  }
+
+  render() {
+    return html`
+      <span id="button">${this.content}</span>
+      <style>
+        :host{
+          display:inline;
+          align-items: center;
+        }
+        #button {
```

Comment:
la parte dello stile va definita in una proprietà style della classe non nel render